### DISCUSSION TITLE
In the VFS, my cosmos kernel can create files, but not directories

### DISCUSSION THREAD

**Discussion by Samdasti** [CosmosOS/Cosmos#2919] [Q&A]
#### Area of Cosmos - What area of Cosmos are we dealing with?
Virtual File System

#### Expected Behaviour - What do you think that should happen?
It should create a folder in "0:\\"

#### Actual Behaviour - What unexpectedly happens?
The folder doesn't get created!

#### Reproduction - How did you get this error to appear?
When i type the command to make a folder, it recognizes it as invalid even if i put the command in the code.

#### Version - Were you using the User Kit or Dev Kit? And what User Kit version or Dev Kit commit (Cosmos, IL2CPU, X#)?
User Kit latest version.

**Comment by Skye310** [comment]
Can you link the code where the issue occurs?

**Reply by Samdasti** [reply]
https://github.com/Samdasti/BasicOS1.0
